src/OpenLoco/src/Ui/Windows/Construction/Common.cpp

```diff
@@ -1476,7 +1476,7 @@ namespace OpenLoco::Ui::Windows::Construction
                     {
                         if (&bridgeList[k] == &bridgeList[bridgeCount])
                         {
-                            _cState->bridgeList[bridgeCount] = i;
+                            bridgeList[bridgeCount] = i;
```

Comment:
This looks like it could've led to issues displaying the bridges dropdown. Is that indeed the case?

Reply:
It doesn't actually make a difference due to them both refering to the same memory.